lib/jsdom/living/helpers/stylesheets.js

```diff
@@ -51,12 +51,25 @@ exports.removeStylesheet = (sheet, elementImpl) => {
 };
 
 // https://drafts.csswg.org/cssom/#create-a-css-style-sheet kinda:
-// - Parsing failures are not handled gracefully like they should be
+// - Parsing failures are now handled gracefully
+// - Like the browser's behaviour, when css.parse() passes trough some invalid CSS
+//   it will try it's best to ignore the invalid parts without blocking the parsing operations
+//   returning a stylesheet with the valid parts only.
+// - css.parse() now receives a funcion that we can use to handle the error as it's second argument
 // - The import rules stuff seems out of place, and probably should affect the load event...
 exports.createStylesheet = (sheetText, elementImpl, baseURLString) => {
   let sheet;
   try {
-    sheet = cssom.parse(sheetText);
```

Comment:
This seems duplicated with the below? Can we either delete the catch (if cssom.parse now never throws) or create a helper function to abstract this?